Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -199,6 +204,15 @@ - (void)setup {
     [self addSubview:self.yearPill];
 }
 
+- (void)setConfiguration:(BPKCalendarConfiguration *)configuration {
+    if (_configuration != configuration) {
+        _configuration = configuration;
+        [self.calendarView registerClass:configuration.cellClass forCellReuseIdentifier:CellReuseId];
```

Comment:
I wonder about the implications of this when registerig a new class with a previously used identifier

Reply:
Yeah i was also thinking maybe the configuration should be set at initialisation. 
`initWithConfig:myConfiguration...`. Might make things easier

Reply:
Sounds good